client/src/app/tabs/panel/tabs/LintingTab.js

```diff
@@ -23,53 +23,53 @@ import WarningIcon from '../../../../../resources/icons/Warning.svg';
 export default function LintingTab(props) {
   const {
     layout,
-    linting,
+    linting: reports,
     onAction,
     onLayoutChanged
   } = props;
 
-  const onClick = (issue) => () => {
-    onAction('showLintError', issue);
+  const onClick = (report) => () => {
+    onAction('showLintError', report);
   };
 
   return <Panel.Tab
     id="linting"
     label="Problems"
     layout={ layout }
-    number={ linting.length }
+    number={ reports.length }
     onLayoutChanged={ onLayoutChanged }
     priority={ 1 }>
-    { linting.length
+    { reports.length
       ? null
       : (
-        <div className={ classnames(css.LintingIssue, 'linting-issue--empty') }>
-          <div className="linting-issue__header">
+        <div className={ classnames(css.LintingTabItem, 'linting-tab-item--empty') }>
+          <div className="linting-tab-item__header">
             <SuccessIcon width="16" height="16" />
-            <span className="linting-issue__label">No problems found.</span>
+            <span className="linting-tab-item__label">No problems found.</span>
           </div>
         </div>
       )
     }
     {
-      sortIssues(linting).map((issue => {
+      sortReports(reports).map((report => {
         const {
           id,
           message
-        } = issue;
+        } = report;
 
-        return <LintingIssue
+        return <LintingTabItem
           key={ `${ id }-${ message }` }
-          issue={ issue }
-          onClick={ onClick(issue) }
+          report={ report }
+          onClick={ onClick(report) }
         />;
       }))
     }
   </Panel.Tab>;
 }
 
-function LintingIssue(props) {
+function LintingTabItem(props) {
```

Comment:
What's wrong with the old name?

Reply:
I mean most of this PR is about name change but I don't see any explanation anywhere.

Reply:
My intention was to reduce the number of names we use to refer to lint reports. So from the UI perspective, there's just the tab and its items. From the linting perspective, there are reports. _Issues_ was yet another name.